README.md

```diff
@@ -13,268 +13,39 @@
-[too-many-secrets](https://github.com/getlantern/too-many-secrets) and must be
-installed to the system's key chain beforehand.
-
-If signing fails, the script will still build the app bundle and dmg, but the
-app bundle won't be signed. Unsigned app bundles can be used for testing but
-should never be distributed to end users.
-
-The background image for the DMG is
-`installer-resources/darwin/dmgbackground.svg`.
-
-### Packaging for Windows
-
-Lantern on Windows is distributed as an installer built with
-[nsis](http://nsis.sourceforge.net/). The installer is built and signed with
-`make docker-package-windows`.
-
-For `make package-windows` to be able to sign the executable, the environment variables
-`SECRETS_DIR` and `BNS_CERT_PASS` must be set to point to the secrets directory
-and the
-[password](https://github.com/getlantern/too-many-secrets/blob/master/build-installers/env-vars.txt#L3)
-of the BNS certificate.  You can set the environment variables and run the
-script on one line, like this:
+You can alternatively skip make entirely and just do:
 
 ```sh
-SECRETS_DIR=$PATH_TO_TOO_MANY_SECRETS BNS_CERT_PASS='***' \
-VERSION=2.0.0-beta1 make package-windows
-```
-
-### Packaging for Ubuntu
-
-Lantern on Ubuntu is distributed as a `.deb` package. You can generate a Debian
-package with:
-
-```sh
-VERSION=2.0.0-beta2 make docker-package-linux
-```
-
-The version string must match the Debian requirements:
-
-https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version
-
-This will build both 386 and amd64 packages.
-
-### Generating all packages
-
-Use the `make packages` task combining all the arguments that
-`package-linux`, `package-windows` and `package-darwin` require.
-
-```sh
-SECRETS_DIR=$PATH_TO_TOO_MANY_SECRETS BNS_CERT_PASS='***' \
-VERSION=2.0.0-beta1 make packages
-```
-
-## Creating releases
-
-### Releasing for QA
-
-In order to release for QA, first obtain an [application token][1] from Github
-(`GH_TOKEN`) and then make sure that [s3cmd](https://github.com/s3tools/s3cmd)
-is correctly configured:
-
-```
-s3cmd --config
-```
-
-Then, create all distribution packages:
-
-```
-[...env variables...] make packages
-```
-
-Finally, use `release-qa` to upload the packages that were just generated to
-both AWS S3 and the Github release page:
-
-```
-VERSION=2.0.0-beta5 make release-qa
-```
-
-### Releasing Beta
-
-If you want to release a beta you must have created a package for QA first,
-then use the `release-beta` task:
-
-```
-make release-beta
-```
-
-`release-beta` will promote the QA files that are currently in S3 to beta.
-
-### Releasing for production
-
-After you're satisfied with a beta version, it will be time to promote beta
-packages to production and to publish the packages for auto-updates:
-
-```
-VERSION=2.0.0-beta5 GH_TOKEN=$GITHUB_TOKEN make release
+git clone https://github.com/getlantern/lantern.git
+cd lantern
+source setenv.bash
+go build -o lantern github.com/getlantern/flashlight/main
```

Comment:
Ah true. A little unfortunate that's required even when just running locally, but that's a different issue! I'll remove that version.